src/calendar/interfaces.ts

```diff
@@ -51,17 +51,17 @@ export interface CalendarProps extends BaseComponentProps {
   /**
    * Used as part of the `aria-label` for today's date in the calendar.
    */
-  todayAriaLabel: string;
+  todayAriaLabel?: string;
 
   /**
    * Specifies an `aria-label` for the 'next month' button.
    */
-  nextMonthAriaLabel: string;
+  nextMonthAriaLabel?: string;
```

Comment:
ditto on these, maybe we should also update where the strings are stored in the message packs for clarity?